modules/starting-a-jupyter-notebook-server.adoc

```diff
@@ -44,6 +44,7 @@ When a new version of a notebook image is released, the previous version remains
 --
 .. From the *Container size* list, select a suitable container size for your server.
 .. Optional: From the *Accelerator* list, select an accelerator. 
+If project-scoped accelerators exist, the *Accelerator* list includes subheadings to distinguish between global accelerators and project-scoped accelerators.
```

Comment:
I don't think a Jupyter notebook server is created in a specific project. So, maybe we can remove this from this file?